src/dev/failed_tests/report.js

```diff
@@ -123,10 +126,10 @@ const updateGithubIssues = (githubClient, issues) => {
       }
     });
 
-    Promise
+    return Promise
       .all(issueOps)
-      .then(() => cb(null, failureCases))
-      .catch(e => cb(e));
+      .then(() => failureCases)
+      .catch(e => e);
```

Comment:
This catch converts the error to the resolve value, it should just be removed so the error can bubble up.